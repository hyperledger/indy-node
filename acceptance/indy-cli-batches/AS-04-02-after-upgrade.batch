#setup environment
pool connect AS-pool
wallet open AS-04-wallet key=testkey
did use V4SGRU86Z58d6TV7PBUe6f
ledger nym did=542MVr22zcHbVyGzaXmbT2
#check that data still available after upgrade
ledger get-schema did=V4SGRU86Z58d6TV7PBUe6f name=Degree version=1.0
ledger get-cred-def schema_id=1 signature_type=CL origin=V4SGRU86Z58d6TV7PBUe6f tag="1"
- ledger get-nym did=542MVr22zcHbVyGzaXmbT5
- ledger get-nym did=542MVr22zcHbVyGzaXmbT2
wallet close
pool disconnect
