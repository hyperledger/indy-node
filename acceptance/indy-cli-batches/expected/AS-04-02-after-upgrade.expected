pool connect AS-pool
[1;32mPool "AS-pool" has been connected[0m

wallet open AS-04-wallet key=testkey
[1;32mWallet "AS-04-wallet" has been opened[0m

did use V4SGRU86Z58d6TV7PBUe6f
[1;32mDid "V4SGRU86Z58d6TV7PBUe6f" has been set as active[0m

ledger nym did=542MVr22zcHbVyGzaXmbT2
[1;32mNym request has been sent to Ledger.[0m
[1;32mMetadata:[0m
+---------------------+---------------------+
| Request ID          | Transaction time    |
+---------------------+---------------------+
| 1527789271458474578 | 2018-05-31 17:54:31 |
+---------------------+---------------------+
[1;32mData:[0m
+------------------------+------+
| Did                    | Role |
+------------------------+------+
| 542MVr22zcHbVyGzaXmbT2 | -    |
+------------------------+------+

ledger get-schema did=V4SGRU86Z58d6TV7PBUe6f name=Degree version=1.0
[1;32mFollowing Schema has been received.[0m
[1;32mMetadata:[0m
+------------------------+-----------------+---------------------+---------------------+
| Did                    | Sequence Number | Request ID          | Transaction time    |
+------------------------+-----------------+---------------------+---------------------+
| V4SGRU86Z58d6TV7PBUe6f | 16              | 1527789271710847095 | 2018-05-31 17:52:57 |
+------------------------+-----------------+---------------------+---------------------+
[1;32mData:[0m
+--------+---------+----------------------------------------------------------------------------+
| Name   | Version | Attributes                                                                 |
+--------+---------+----------------------------------------------------------------------------+
| Degree | 1.0     | "postgrad","first_name","last_name","birth_date","undergrad","expiry_date" |
+--------+---------+----------------------------------------------------------------------------+

ledger get-cred-def schema_id=1 signature_type=CL origin=V4SGRU86Z58d6TV7PBUe6f tag="1"
[1;32mFollowing Credential Definition has been received.[0m
[1;32mMetadata:[0m
+------------------------+-----------------+---------------------+---------------------+
| Identifier             | Sequence Number | Request ID          | Transaction time    |
+------------------------+-----------------+---------------------+---------------------+
| V4SGRU86Z58d6TV7PBUe6f | 17              | 1527789271740916519 | 2018-05-31 17:52:57 |
+------------------------+-----------------+---------------------+---------------------+
[1;32mData:[0m
+----------------------------------------------------------------+
| Primary Key                                                    |
+----------------------------------------------------------------+
| {n:"1",r:{"age":"4","name":"5"},rctxt:"6",rms:"3",s:"2",z:"7"} |
+----------------------------------------------------------------+

- ledger get-nym did=542MVr22zcHbVyGzaXmbT5
[1;32mFollowing NYM has been received.[0m
[1;32mMetadata:[0m
+-----------------+---------------------+---------------------+
| Sequence Number | Request ID          | Transaction time    |
+-----------------+---------------------+---------------------+
| 20              | 1527789271777541476 | 2018-05-31 17:52:58 |
+-----------------+---------------------+---------------------+
[1;32mData:[0m
+------------------------+------------------------+------+
| Identifier             | Dest                   | Role |
+------------------------+------------------------+------+
| V4SGRU86Z58d6TV7PBUe6f | 542MVr22zcHbVyGzaXmbT5 | -    |
+------------------------+------------------------+------+

- ledger get-nym did=542MVr22zcHbVyGzaXmbT2
[1;32mFollowing NYM has been received.[0m
[1;32mMetadata:[0m
+-----------------+---------------------+---------------------+
| Sequence Number | Request ID          | Transaction time    |
+-----------------+---------------------+---------------------+
| 23              | 1527789271796959032 | 2018-05-31 17:54:31 |
+-----------------+---------------------+---------------------+
[1;32mData:[0m
+------------------------+------------------------+------+
| Identifier             | Dest                   | Role |
+------------------------+------------------------+------+
| V4SGRU86Z58d6TV7PBUe6f | 542MVr22zcHbVyGzaXmbT2 | -    |
+------------------------+------------------------+------+

wallet close
[1;32mWallet "AS-04-wallet" has been closed[0m

pool disconnect
[1;32mPool "AS-pool" has been disconnected[0m

[1;32mGoodbye...[0m
